src/test/java/org/altbeacon/beacon/distance/ModelSpecificDistanceCalculatorTest.java

```diff
@@ -58,4 +52,60 @@ public void testSelectsNexus4OnExactMatch() {
         assertEquals("should be Nexus 4", "Nexus 4", distanceCalculator.getModel().getModel());
     }
 
+	@Test
+	public void testConcurrentModificationException() {
+		org.robolectric.shadows.ShadowLog.stream = System.err;
+
+		final Context applicationContext = ShadowApplication.getInstance().getApplicationContext();
+
+		final AndroidModel model = new AndroidModel("4.4.2", "KOT49H", "Nexus 4", "LGE");
+		final ModelSpecificDistanceCalculator distanceCalculator =
+				new ModelSpecificDistanceCalculator(applicationContext, null, model);
+
+		Runnable runnable2 = new Runnable() {
+			@Override
+			public void run() {
+				try {
+					while (true) {
+						distanceCalculator._buildModelMapTest("{\n" +
+															  "  \"models\":\n" +
+															  "  [\n" +
```

Comment:
I completely missed that!  Any idea why the auto formatter is indenting it so much?  I bet I won't be the only person who misses this because it is off the screen to the right.